dask_cloudprovider/providers/azure/azureml.py

```diff
@@ -0,0 +1,804 @@
+from azureml.core import Experiment, RunConfiguration, ScriptRunConfig
+from azureml.core.compute import AmlCompute
+from azureml.train.estimator import Estimator
+from azureml.core.runconfig import MpiConfiguration
+
+import time, os, socket, subprocess, logging
+import pathlib
+import threading
+
+import dask
+from distributed.deploy.cluster import Cluster
+from distributed.core import rpc
+from distributed.utils import (
+    LoopRunner,
+    log_errors,
+    ignoring,
```

Comment:
distributed is no longer using `ignoring` and instead using `from contextlib import suppress` -- see https://github.com/dask/distributed/pull/3819